modules/oxide/src/less/theme/components/form/color.less

```diff
@@ -29,14 +29,15 @@
       border-width: 1px;
       box-shadow: none;
       box-sizing: border-box;
-      cursor: pointer;
       height: @textfield-height - 12px;
       position: absolute;
       top: 6px;
       width: $height;
 
-      &:focus {
+      &:hover:not([aria-disabled=true]),
```

Comment:
Should we be using a class here instead of relying on this attribute?

Reply:
Moving this to a separate ticket